### ISSUE TITLE
Duplication of Content-Type header

### ISSUE THREAD

**Issue by eddig** [fortio/fortio#291]
**Description**
There are duplications of `Content-Type` header in the requests that are generated by Fortio. The number of duplicated headers is corresponding to the defined number of connections.

**How to reproduce**
1. Run `fortio load -content-type 'application/json' -payload '{"query":"{__schema { types { kind } }}"}' -c 3 -qps 0 -n 1 -loglevel Debug https://31zrkwkkv.lp.gql.zone/graphql`
2. In the debug output will be three requests with different set of headers:
```
Content-Type: application/json
```
```
Content-Type: application/json
Content-Type: application/json
```
```
Content-Type: application/json
Content-Type: application/json
Content-Type: application/json
```

**What expected**
Only one `Content-Type: application/json` header per request

**Comment by ldemailly** [comment]
oh nice bug ! thank you for finding and reporting it !
the content type is added each time in each parallel query, so the first one has 1, second one 2, ... fourth 4... 

let me fix this !

**Comment by ldemailly** [comment]
fixed in 1.2.0 - thanks !
